test/unit/test_modprobe.py

```diff
@@ -125,7 +125,8 @@ def test_error_invalid_boolean_value_simple_format(self):
             self.new_cluster()
 
     def test_multiple_os_family_merge(self):
-        supported_os_families = list(packages.get_associations_os_family_keys())
+        # Exclude legacy 'rhel' family for merge behavior since no nodes run plain 'rhel'
+        supported_os_families = [f for f in packages.get_associations_os_family_keys() if f != 'rhel']
```

Comment:
It seems that if we remove `rhel` from the list, we do not need to exclude it here